vulcanai2/models/layers.py

```diff
@@ -146,6 +150,9 @@ def __init__(self, in_features, out_features,
         # Activation/Non-Linearity
         if activation is not None:
             self.add_module('_activation', activation)
+            if isinstance(activation, nn.SELU):
+                self.initializer = selu_init_
```

Comment:
can we call it `weight_init` instead of initializer so it matches the `bias_init` naming convention?